assignments/prototypes.js

```diff
@@ -15,13 +15,33 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject(atti){
+  Humanoid.call(this, atti);
+}
+
+CharacterStats.prototype = Object.create(GameObject.prototype);
+
+GameObject.prototype.destroy = function() {
+  return (`${this.name} was removed from the game.`)
+}
+
 
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(attrib){
+  Humanoid.call(this, attrib);
+}
+
+Humanoid.prototype = Object.create(CharacterStats.prototype);
```

Comment:
Again this here is backwards. You need to run `<parent>.call(this, <attributes of child>)`. So here it should be called on the `GameObject` here so that it has access to all of the parents properties. (the parents template ie, createdAt, dimensions) and you can add your own the parent doesn't have.